java/build.gradle

```diff
@@ -20,7 +20,6 @@ dependencies {
 sourceSets {
     test {
         java {
-            srcDir 'example'
```

Comment:
Is there a separate way that we could add the examples into this build file as optional? libsbp's current docker doesn't include mvn. Or maybe there would be value in including a separate gradle file just for building the examples?